src/main/java/com/example/springbootrestexample/mongoDB/AuthorCollectionongoJavaDriver.java

```diff
@@ -0,0 +1,58 @@
+package com.example.springbootrestexample.mongoDB;
+
+import com.example.springbootrestexample.models.Author;
+import com.mongodb.BasicDBObject;
+import com.mongodb.DBCollection;
+import com.mongodb.DBCursor;
+import com.mongodb.DBObject;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class AuthorCollectionongoJavaDriver extends MongoDBConnection {
```

Comment:
The other implementation is nicer

Reply:
Yes, it was only for testing and viewing the 2 ways of using MongoDB (this apply not for Spring Applications). I'll leave it only for learning purpose.

Reply:
Ok